Feature/#2 add player object

omer21104 opened the pull request:
player object created with a lives counter as data member.
basic functionality added as well.
put in checks in place to assert x and y values are positive, might need to change later.

brandonvl commented on IntoSpace/Player.h:
```diff
@@ -0,0 +1,22 @@
+{
+private:
+	int lives; 
+	float x, y, velX, velY;
```
Alter variables to m_ prefix variables are discussed

brandonvl commented on IntoSpace/Player.h:
```diff
@@ -0,0 +1,22 @@
+public:
+	Player(float x = 0, float  y = 0, float  velX = 0, float  velY = 0);
+	~Player();
+	void set_x(float x);
```
rename to PascalCasing formatting as agreed

brandonvl commented on IntoSpace/Player.h:
```diff
@@ -0,0 +1,22 @@
+#pragma once
```
not all compilers implement #pragma yet, use #ifndef guarding instead.

brandonvl commented on IntoSpace/Player.h:
```diff
@@ -0,0 +1,22 @@
+#pragma once
+const int DEFAULT_LIVES = 3;
```
Refactor constant to constants.h file

omer21104 committed:
#2: Added Player class with basic functionality.

Added Player class with basic functions,
set_x and set_y methods assert positive values for now.

omer21104 committed:
#2: Changing naming convention

changed naming convention and added constants header file for player object
